app/components/button_component.rb

```diff
@@ -8,7 +8,8 @@ def initialize(
     method: :put,
     confirm: nil,
     disabled: false,
-    classes: nil
+    classes: nil,
+    turbo_stream: nil
```

Comment:
Should this be `false` instead of `nil`? Doesn't make a functional difference right now, but feels more correct.

Reply:
`false` feels better for sure.